store.py

```diff
@@ -85,13 +86,11 @@ def read_all(self):
         """Read all parts from the database."""
```

Comment:
should we annotate this with a '-> dict' hint?

Reply:
Absolutely, will add that in a minute